haystack_experimental/dataclasses/streaming_chunk.py

```diff
@@ -5,8 +5,7 @@
 from typing import Awaitable, Callable, Optional, Union
 
 from haystack.dataclasses import StreamingChunk
-
-from haystack_experimental.util import is_callable_async_compatible
+from haystack.utils.asynchronous import is_callable_async_compatible
```

Comment:
Hey @davidsbatista I think it's also safe to remove the `select_streaming_callback` function since I don't think it's being used anymore after your removals.

Reply:
True, thanks! I'll remove it.